wiki/en/en-Command-Line-Options.md

```diff
@@ -42,6 +42,7 @@ You can see all possible options your version supports by starting Jamulus with
 |    `-n` |`--nogui`          | Disable GUI | (client and server) |
 |    `-o` |`--serverinfo`     | Server location details in the format: <br/>`[name];[city];[locale value]` (see [values](https://doc.qt.io/qt-5/qlocale.html#Country-enum))| (public servers only) |
 |    `-p` |`--port`           | Local UDP port number. Default is 22124 | (server only) |
+|    `-P` |`--delaypan`  |  Start with delay panning enabled | (server only) |
```

Comment:
Maybe add a link to Server-Win-Mac.md?

Reply:
I was hoping they'd just Google it  :-)

Reply:
Hmm. Not sure if that's really the case. But who knows?

Reply:
It's a grey area I guess. We explain it in the GUI because it's visible there and isn't widely known about, but it's not unique to Jamulus in the same way as reverb is, and attempting to explain those isn't feasible. But I guess as we have it in the docs we should link it.